packages/capnpc-ts/src/ast-creators.ts

```diff
@@ -68,6 +68,27 @@ export function createNestedNodeProperty(node: s.Node): ts.PropertyDeclaration {
 
 }
 
+export function createObjectLiteral(properties: { [key: string]: ts.Expression }): ts.Expression {
```

Comment:
This is fine, actually just a note to myself: `capnpc-ts` is in the middle of a pretty heavy upgrade, so this will require careful merging. Should not attempt to rebase my working branch from here.